Source: Santiago M. Mola <coldwind@gentoo.org>
Upstream: The 'locate' part is merged.
Reason: Support Python 2.6 and stop using locate to find Python headers.
--- libtorrent-rasterbar-0.14.1.orig/m4/ax_python.m4	2008-12-28 15:07:42.000000000 +0100
+++ libtorrent-rasterbar-0.14.1/m4/ax_python.m4	2008-12-28 18:48:54.000000000 +0100
@@ -58,13 +58,13 @@
 AC_DEFUN([AX_PYTHON],
 [AC_MSG_CHECKING(for python build information)
 AC_MSG_RESULT([])
-for python in python2.5 python2.4 python2.3 python2.2 python2.1 python; do
+for python in python2.6 python2.5 python2.4 python2.3 python2.2 python2.1 python; do
 AC_CHECK_PROGS(PYTHON_BIN, [$python])
 ax_python_bin=$PYTHON_BIN
 if test x$ax_python_bin != x; then
    AC_CHECK_LIB($ax_python_bin, main, ax_python_lib=$ax_python_bin, ax_python_lib=no)
    AC_CHECK_HEADER([$ax_python_bin/Python.h],
-   [[ax_python_header=`locate $ax_python_bin/Python.h | sed -e s,/Python.h,,`]],
+   [[ax_python_header=/usr/include/$ax_python_bin]],
    ax_python_header=no)
    if test $ax_python_lib != no; then
      if test $ax_python_header != no; then
